robotbase/src/main/java/com/team2357/frc2022/Constants.java

```diff
@@ -12,4 +12,19 @@
  * <p>It is advised to statically import this class (or one of its inner classes) wherever the
  * constants are needed, to reduce verbosity.
  */
-public final class Constants {}
+public final class Constants {
+
+    // CAN IDs
+    public static final int DRIVE_MOTOR_LEFT_1 = 11;
+    public static final int DRIVE_MOTOR_RIGHT_1 = 12;
+    public static final int DRIVE_MOTOR_LEFT_2 = 13;
+    public static final int DRIVE_MOTOR_RIGHT_2 = 14;
+    public static final int GYRO_ID = 5;
+
+    //Controller Constants
+    public static final int DRIVE_CONTROLLER_PORT = 0;
+    public static final double DRIVE_CONTROLLER_DEADBAND = 0.1;
+    // Encoder Constants
+    public static final int ENCODER_DISTANCE_PER_PULSE = 256;
```

Comment:
This should probably be more descriptive as we'll be adding other encoders later. Also, is this actual distance? If so, what units?

Reply:
This was just a dummy constant for now, but I added meters.